products.css

```diff
@@ -0,0 +1,123 @@
+@import url(main.css);
+.product-page
+{
+    display: flex;
+}
+.sidebar
+{
+    margin-left: 4rem;
+    width: 16rem;
+    box-shadow: 0px 0px 11px 0px var(--boxshawdow);
+    padding: 10px;
+}
+
+ul
+{
+    list-style: none;
+}
+.filters
+{
+    display: flex;
+    justify-content: space-between;
+    width: 14rem;
+}
+.filter-content
+{
+    font-size: 1.6rem;
+    text-decoration: underline;
+    font-weight: 100;
+    color:var(--black);
+    margin: 1rem 0;
+    padding: 5px;
+}
+.Price-Content
+{
+    display: flex;
+    justify-content: space-between;
+    width: 14rem;
+}
+input[type='range']
+{
+    background-color: var(--black);
+    width: 14rem;
+}
+h2{
```

Comment:
Change line of h2 like you have given spaces to previous elements

Reply:
Done